src/components/collection-progress-group/helpers/parse-collection-progress.ts

```diff
@@ -34,25 +34,12 @@ function parseCollectionProgress(
 		completedTutorialCount,
 		tutorialCount,
 	})
-	/**
-	 * Status label
-	 */
-	let statusLabel: string
```

Comment:
Have moved `statusLabel` logic into `collection-progress-status-section`, it felt more intuitive there, and simplified the props API of that component.